client/main.lua

```diff
@@ -1683,6 +1683,7 @@ function CreateBlips()
 
```

Comment:
Hidden value declaration on config without implementation in client/main.lua

https://github.com/lockdownstudio/esx_advancedgarage/blob/270bed0fb26cb858e654720ae5c12ab363245283/config.lua#L88

https://github.com/lockdownstudio/esx_advancedgarage/blob/270bed0fb26cb858e654720ae5c12ab363245283/config.lua#L129

https://github.com/lockdownstudio/esx_advancedgarage/blob/270bed0fb26cb858e654720ae5c12ab363245283/config.lua#L328

Either delete the hidden value or create an implementation on client/main.lua